testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -55,4 +62,14 @@ public static String getPathFromClassPathResource(String resourcePath)
                 // and mounted from there.
                 .getResolvedPath();
     }
+
+    // utility method for setting exposed port to default values, like 5432 (postgres)
+    // for more comfortable usages in local environments for direct attach
+    public static void configureFixedExposedPortForLocalEnv(GenericContainer<?> container, int port)
+    {
+        if (!TESTCONTAINERS_IS_CI_ENV) {
+            container.withCreateContainerCmdModifier(cmd -> cmd.withHostConfig(
```

Comment:
Since `cmd` is a builder I think this is safe to do even if the container already provided it's own `withCreateContainerCmdModifier`?

Reply:
I think yes looks, like we add all modifiers to some internal collection, and then execute all of them one by one, so we can call this method multiple times